tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,22 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func getVcpuMask(pod *k8sv1.Pod, emulator, cpu string) (output string, err error) {
+	BinBash := "/bin/bash"
```

Comment:
nit: This could be a const and start with a lowercase letter.

Reply:
Done